app-misc/qlcplus/qlcplus-4.14.1.ebuild

```diff
@@ -39,31 +36,28 @@ RDEPEND="
 IDEPEND="
 	dev-util/desktop-file-utils
 "
-DEPEND="${RDEPEND}
-	dev-qt/qttest:5
+DEPEND="
+	${RDEPEND}
 "
 
 src_prepare() {
 	default
 
-	sed -e "/UDEVRULESDIR/s:/etc/udev/rules.d:$(get_udevdir)/rules.d:" \
-		-i variables.pri || die
+	sed -e "s|lib/${CMAKE_C_LIBRARY_ARCHITECTURE}|$(get_libdir)|g" \
```

Comment:
maybe we should advertise GNUInstallDirs cmake module to upstream to avoid such hacks ...

Reply:
True, I will talk to the devs or prepare a PR myself. Thanks for the hint